datashuttle/datashuttle_class.py

```diff
@@ -868,10 +868,36 @@ def setup_ssh_connection(self) -> None:
 
         if verified:
             ssh.setup_ssh_key(self.cfg, log=True)
+            # 449
+            confirmed_path = (
+                input(
+                    f"Confirm the full central path (default: {self.cfg['central_path']}): "
+                )
+                or self.cfg["central_path"]
+            )
+            self.cfg["central_path"] = confirmed_path
+            if not self._check_write_permissions(confirmed_path):
+                raise PermissionError(
+                    f"Cannot write to the central path: {confirmed_path}"
+                )
+
             self._setup_rclone_central_ssh_config(log=True)
 
         ds_logger.close_log_filehandler()
 
+    # 449
+    def _check_write_permissions(self, path) -> bool:
```

Comment:
This is cool, I think this function can go into `ssh.py` and be merged with `run_ssh_command`. It is nice to encapsulate the code for writing commands over SSH but as we do this [elsewhere](https://github.com/neuroinformatics-unit/datashuttle/blob/166f8620199b6392df43d0889a8b5e34b263d3a0/datashuttle/utils/ssh.py#L66) in a way that would make it difficult to use `run_ssh_command`.  For now we can use `client.exec_command` more flexibly from within other functions and refactor to centralise under a single function at a later time.